packages/editor/tests/oas2.spec.ts

```diff
@@ -0,0 +1,255 @@
+import {
+  OasV2Editor,
+  SpecTreeNode,
+  SpecTreeRequestBodyParam,
+  SpecTreeNodeParam,
+  TreeParser
+} from '../src';
+import $RefParser from '@apidevtools/json-schema-ref-parser';
+import { load } from 'js-yaml';
+import jsonPath from 'jsonpath';
+import 'chai/register-should';
+import { OpenAPIV2 } from '@har-sdk/types';
+import { OASValidator } from '@har-sdk/validator';
+import chaiAsPromised from 'chai-as-promised';
+import { use } from 'chai';
+import { readFileSync } from 'fs';
+import { resolve } from 'path';
+
+use(chaiAsPromised);
+
+describe('oas parser', () => {
+  const sourcePath = './tests/oas2-sample1.yaml';
+  const source = readFileSync(resolve(sourcePath), 'utf-8');
+  const expectedResultPath = './tests/oas2-sample1.result.json';
+
+  describe('dereference', () => {
+    it('should dereference all $refs', async () => {
+      const parser = new $RefParser();
+      const api = await parser.dereference(sourcePath);
+      JSON.stringify(api).should.not.contain('$ref');
+    });
+  });
+
+  describe('validate basic oas2 sample', () => {
+    it('should be validated with no errors', async () => {
+      const verifyResultPromise = new OASValidator().verify(
+        load(source, { json: true }) as OpenAPIV2.Document
+      );
+
+      return verifyResultPromise.should.eventually.deep.eq({
+        errors: [],
+        valid: true
+      });
+    });
+  });
+
+  describe('oas v2 tree parser', () => {
+    const openApiParser: TreeParser = new OasV2Editor();
+    const resultTree = JSON.parse(
+      readFileSync(resolve(expectedResultPath), 'utf-8')
+    );
+
+    it('should be exception on invalid syntax', () =>
+      openApiParser
+        .setup('{')
+        .should.be.rejectedWith(
+          Error,
+          /Bad Swagger\/OpenAPI V2 specification/
```

Comment:
As far as I know you can use a string as well
```suggestion
          'Bad Swagger/OpenAPI V2 specification'
```